src/cpr_sdk/yql_builder.py

```diff
@@ -58,26 +58,16 @@ def build_search_term(self) -> str:
         elif self.sensitive:
             return """
                 (
-                    {"targetHits": 1000} weakAnd(
-                        family_name contains(@query_string),
-                        family_description contains(@query_string),
-                        text_block contains(@query_string)
-                    )
+                    userInput(@query_string)
```

Comment:
This might be something that I'm missing when looking at the docs, but I'm struggling to get my head around how userInput 'knows' which fields to look at, do you have a clearer understanding of this you could help me with?

Reply:
yes - i think it's the default fields specified [in the schema](https://github.com/climatepolicyradar/navigator-infra/blob/6a6460bcd8d5e863df1ef846b856c0a239b55c13/vespa/pods_research/schemas/family_document.sd#L108)

Reply:
Ah that makes sense